Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,73 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * Object used to define bounds limiting the {@link @Camera#position} when in 3D mode.
+     *
+     * {@link @Camera#cameraLimiter}
+     *
+     * Used by a {@link Camera} to limit the camera's position.
+     *
+     * @alias CameraLimiter
+     * @constructor
+     *
+     * @param {AxisAlignedBoundingBox|BoundingSphere|OrientedBoundingBox} [options.boundingObject] The BoundingObject used to restrict camera positions of the {@link Camera} holding this CameraLimiter.
```

Comment:
This sholdn't be an `options` parameter if there's only one.  We use an `options` object when a function takes a ton of different parameters to make it more concise.  See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#options-parameters

Also, is this parameter required?

Reply:
Do we need a separate class for this?  There's hardly any logic in this class, why not just put it all in `Camera`?

Reply:
originally this class contain hpr constraints and a bounding object - which is why the options parameter was needed - we're only splitting out into this branch [ie it just has bounding object instead of bobject and hpr] so we can try to merge with just it for now and remove the current jitter issue from the hpr constraint in another issue